src/components/navigation/desktop/TypeAheadDropdown.tsx

```diff
@@ -21,7 +21,10 @@ export const TypeAheadDropdown = (props: Props) => {
         props.focusChange(false);
       }}
     >
-      <VmComponent src={components.search.typeAheadDropdown} props={{ term: props.term }} />
+      <VmComponent
+        src={components.search.typeAheadDropdown}
+        props={{ term: props.term, closeSearch: props.focusChange }}
```

Comment:
Let's keep our naming more consistent to avoid confusion. Instead of `closeSearch`, let's do:

```
focusChange: props.focusChange
```

Reply:
This has been fixes. Thank you for the review.